packages/commerce-sdk-react/src/hooks/ShopperCustomers/types.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {QueryParams} from '../types'
+
+export interface ShopperCustomerParams extends QueryParams {
```

Comment:
I think that most of these types can now be derived from `commerce-sdk-isomorphic`.